pkg/devfile/generator/generators_test.go

```diff
@@ -280,12 +280,13 @@ func TestGetContainers(t *testing.T) {
 			// to set up the prestartevent and apply command for init container
 			mockGetCommands := mockDevfileData.EXPECT().GetCommands(common.DevfileOptions{})
 			mockGetCommands.Return(applyCommands, nil).AnyTimes()
-			preStartEvents := v1.Events{
+			events := v1.Events{
 				DevWorkspaceEvents: v1.DevWorkspaceEvents{
 					PreStart: tt.eventCommands,
+					PostStop: tt.eventCommands,
```

Comment:
last small nit: can you make a separate command for PostStop? Both the events are referencing the same command, but want to make sure we are testing if its removed properly

Reply:
updated